src/chttpd/src/chttpd_node.erl

```diff
@@ -137,7 +137,7 @@ handle_node_req(#httpd{path_parts=[_, Node | PathParts],
     {_, Query, Fragment} = mochiweb_util:urlsplit_path(RawUri),
     NewPath0 = "/" ++ lists:join("/", [couch_util:url_encode(P) || P <- PathParts]),
     NewRawPath = mochiweb_util:urlunsplit_path({NewPath0, Query, Fragment}),
-    MaxSize =  config:get_integer("httpd", "max_http_request_size", 4294967296),
```

Comment:
Previously we expected an integer back but now we'd return a list (string) value back if the file has the value set. We could technically tell from the value of the default variable which type we expect back - if the default is an integer we'd want to do a `config:get_integer/3`, if the default is a boolean a `config:get_boolean/3`. So it would kind of work, but it would be inconsistent with how the rest of the config code looks. So perhaps the best option would be to have a set of `chttd_util:get_chttpd_config_integer/3` and `chttpd_util:get_chttpd_config_boolean/3` functions.

Reply:
Added get_chttpd_config_integer/3 and get_chttpd_config_boolean/3 functions.